.github/workflows/nightly.yml

```diff
@@ -0,0 +1,42 @@
+name: Nightly tests
+
+on:
+    workflow_dispatch: # note: if started manually, it won't run all matrix
+    schedule:
+        - cron: "0 3 * * *"  #  Runs at 03:00 (3 AM) UTC every day
```

Comment:
Is it good time for you guys as well? We should try to make it as less hurting as possible, for us if everything ends by 7 AM we are good. Let's calculate from there backwards.

Reply:
UTC Midnight occurs during our workday. This job will completely destroy the CI queue by adding ~200 jobs.
https://github.com/Yury-Fridlyand/valkey-glide/actions/runs/10641495001
* There are 120 jobs for python (60 + 60, because PubSub IT are in separate job)
* 24 for java (will be 48 once I split pubsub too)
* 24 for C# (could be deactivated)
* 12 for go
* 100 for node
* 20 for glide core lib

I don't know how much time this might take. It depends on number of runners available on valkey org and on number of other nightly jobs (valkey repo has also nightly CI jobs)